apps/web/src/pages/ApplicantDashboardPage/ApplicantDashboardPage.tsx

```diff
@@ -33,6 +40,91 @@ export const ApplicantDashboardPage_Fragment = graphql(/* GraphQL */ `
     poolCandidates {
       ...ApplicationsProcessesTaskCard
     }
+    lookingForEnglish
+    lookingForFrench
+    lookingForBilingual
+    estimatedLanguageAbility {
+      value
+      label {
+        en
+        fr
+      }
+    }
+    firstOfficialLanguage {
+      value
+      label {
+        en
+        fr
+      }
+    }
+    secondLanguageExamCompleted
+    secondLanguageExamValidity
+    writtenLevel {
+      value
+      label {
+        en
+        fr
+      }
+    }
+    comprehensionLevel {
+      value
+      label {
+        en
+        fr
+      }
+    }
+    verbalLevel {
+      value
+      label {
+        en
+        fr
+      }
+    }
+    userSkills {
+      id
+      skill {
+        id
+        key
+        name {
+          en
+          fr
+        }
+        category {
+          value
+          label {
+            en
+            fr
+          }
+        }
+      }
+    }
```

Comment:
I beleive just `id` is needed here since it looks like we are just checking to see if at least one exists.

Reply:
7e32d38bf7cf4cf48ce3f405234ab4a68b4fb155